src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable arrow-body-style */
+
+// Types
+const CHECK_STATUS = 'CHECK_STATUS';
+
+// Constants
+const defaultState = [];
+
+// Reducer
+export default function categoriesReducer(state = defaultState, action) {
+  switch (action.type) {
+    case CHECK_STATUS:
+      return console.log('Under Construction');
```

Comment:
- [ ] [OPTIONAL] Kindly remove this `console` statement as it is only meant for debugging. This sometimes makes the application slower and it is also not a good practice to always leave it in your codebase. You can return the 'Under Construction' statement without using `console.log`